src/render.rs

```diff
@@ -62,26 +63,38 @@ pub(crate) fn cosmic_edit_redraw_buffer(
         mut x_offset,
         mode,
         mut placeholder_opt,
+        password_opt,
     ) in &mut cosmic_edit_query.iter_mut()
     {
-        if !editor.0.buffer().redraw() {
+        if !cosmic_editor.0.buffer().redraw() {
             continue;
         }
 
+        let current_text = cosmic_editor.get_text();
+
+        // intercept text for password inputs
+        if password_opt.is_some() && !current_text.is_empty() {
+            cosmic_editor.set_text(
+                crate::CosmicText::OneStyle("•".repeat(current_text.len())),
```

Comment:
what do you think about making it configurable? Password component will keep "•" char to replace chars

Reply:
Yeah sounds good, I like the idea of a password field that blocks with emoji and this would allow it